### ISSUE TITLE
Error while uploading bulk user (csv) in snipe IT

### ISSUE THREAD

**Issue by sobinmathew** [snipe/snipe-it#5188]
@snipe #### Expected Behavior (or desired behavior if a feature request)

csv should be uploaded without error,

-----

importing users by CSV, no issues if the CSV contains 3 users or below, but we are getting an error while uploading bulk user (more than 4 users in CSV) 

CSV contains email address for activating the user

note: csv getting successfully uploaded without  email address 

Error message : Whoops, looks like something went wrong.

-----

#### Please confirm you have done the following before posting your bug report:

- [ yes] I have enabled debug mode 
- [yes ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? : Fresh
- Version of Snipe-IT you're running Version : v4.1.10 - build 3249
- Version of PHP you're running : 5.6.33
- Version of MySQL/MariaDB you're running  : 10.1.30-MariaDB - mariadb.org binary distribution
- What OS and web server you're running Snipe-IT on : Windows 8.1
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by sobinmathew** [comment]
@snipe  PLEASE HELP

**Comment by jaison4u777** [comment]
I am Also Facing same issue.Please Help

**Comment by sobinmathew** [comment]
![settigs](https://user-images.githubusercontent.com/37208514/37444570-5b72833c-2838-11e8-8f82-03bef40ba5b3.JPG)

**Comment by sobinmathew** [comment]
@snipe thanks for your response :) 

We have enabled debugged mode and found the error ...

![4bef6eff](https://user-images.githubusercontent.com/37208514/37581382-5b67521c-2b6e-11e8-912f-2e536ecea6c4.PNG)

using this error we have searched for the solution and fixed it.

the issue was with the ini file, default the range was set to 30 sec

Solution:
locate the file [XAMPP Installation Directory]\php\php.ini (e.g. C:\xampp\php\php.ini)
open php.ini in Notepad or any Text editor
locate the line containing max_execution_time and
increase the value from 30 to some larger number (e.g. set: max_execution_time = 90)
then restart Apache web server from the XAMPP control panel

Thank you very much..
